src/PayID/pay-id-components.ts

```diff
@@ -1,6 +1,8 @@
 // Disable multiple classes to accommodate the switch to idiomatic style naming.
 // TODO(keefertaylor): Remove this when migration is complete.
-/* eslint-disable  max-classes-per-file */
+/* eslint-disable  max-classes-per-file --
+ * We only export two classes in this file because one is deprecated and will eventually be removed.
+ */
```

Comment:
The basic format for an ESLint comment is just requiring two hyphens `--` followed by the comment.

You can do this all on one line, but for longer comments, I thought the multiline style I'm using here was reasonable.